tests/functional/test_reinstall.py

```diff
@@ -190,7 +193,7 @@ async def test_cylc_reinstall_files(
         ),
         (
             'opt/rose-suite-cylc-install.conf', (
-                '# This file records CLI Options.\n\n'
+                '# This file records CLI Options.{version_info}\n'
```

Comment:
This isn't a format string.

The test still passes, so the `version_info` is presumably not require here?